scripts/edit-logs.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+
+path=''
+directory='false'
```

Comment:
```suggestion
directory=false
```
no need to quote anything that is blank or a word (convention)